js/react_native/android/src/main/java/ai/onnxruntime/reactnative/OnnxruntimeModule.java

```diff
@@ -255,16 +255,18 @@ public WritableMap run(String key, ReadableMap input, ReadableArray output, Read
           throw new Exception("Can't find input: " + inputName);
         }
 
-        if (inputMap.getType("data") != ReadableType.String || inputMap.getType("data") != ReadableType.Array) {
-          // NOTE:
-          //
-          // tensor data should always be a BASE64 encoded string.
-          // This is because the current React Native bridge supports limited data type as arguments.
-          // In order to pass data from JS to Java, we have to encode them into string.
-          //
-          // see also:
-          //   https://reactnative.dev/docs/native-modules-android#argument-types
-          throw new Exception("Non string type of a tensor data is not allowed");
+        if (inputMap.getType("data") != ReadableType.String) {
+          if(inputMap.getType("data") != ReadableType.Array){
```

Comment:
I think `if (inputMap.getType("data") != ReadableType.String && inputMap.getType("data") != ReadableType.Array)` should work?

Reply:
Fixed a simple mistake.